react-stopwatch/my-app/src/StopWatch.tsx

```diff
@@ -0,0 +1,38 @@
+import './StopWatch.css';
+import { useState } from 'react';
+import { FaPause, FaPlay } from 'react-icons/fa';
+
+let StopWatchIcon = FaPlay;
+
+export function StopWatch() {
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout>();
+  const [elapsedSeconds, setElapsedSeconds] = useState(0);
+
+  return (
+    <>
+      <div className="stopwatch-face">
+        <div className="stopwatch-seconds" onClick={() => setElapsedSeconds(0)}>
+          {elapsedSeconds}
+        </div>
+      </div>
+      <StopWatchIcon
+        className="stopwatch-icon"
+        onClick={() => {
```

Comment:
This is a pretty long function. It would make cleaner code if you put this into a separate function.